xarray_sentinel/sentinel1.py

```diff
@@ -271,6 +271,11 @@ def open_dataset(
     ancillary_data_paths = esa_safe.get_ancillary_data_paths(
         manifest_path, product_files
     )
+    if drop_variables:
```

Comment:
```suggestion
    if drop_variables is not None:
```
To avoid `drop_variables=0` and `drop_variables=False` not triggering the waring.